packages/dbml-core/src/export/PostgresExporter.js

```diff
@@ -204,13 +204,22 @@ class PostgresExporter {
     const commentArr = comments.map((comment) => {
       let line = 'COMMENT ON';
 
-      if (comment.type === 'column') {
-        const field = model.fields[comment.fieldId];
-        const table = model.tables[field.tableId];
-        const schema = model.schemas[table.schemaId];
-
-        line += ` COLUMN ${shouldPrintSchema(schema, model)
-          ? `"${schema.name}".` : ''}"${table.name}"."${field.name}" IS '${field.note}'`;
+      switch (comment.type) {
+        case 'table': {
+          const table = model.tables[comment.tableId];
+          const schema = model.schemas[table.schemaId];          
+          line += ` TABLE ${shouldPrintSchema(schema, model)
+            ? `"${schema.name}".` : ''}"${table.name}" IS '${table.note}'`;
+          break; 
+        }
+        case 'column': {
+          const field = model.fields[comment.fieldId]; 
+          const table = model.tables[field.tableId];
+          const schema = model.schemas[table.schemaId];
+          line += ` COLUMN ${shouldPrintSchema(schema, model)
```

Comment:
Should get table and schema before the switch statement to avoid repeated code here.
```
const table = model.tables[field.tableId];
const schema = model.schemas[table.schemaId];

switch (comment.type) { ...
```